src/index.js

```diff
@@ -1,48 +1,49 @@
-/* eslint-disable no-unused-vars */
-import _ from 'lodash';
 import './style.css';
+import {
+  getLocalStorage,
+  addToDoTask,
+  deleteToDoTask,
+  editToDoTask,
+} from './addremove.js';
+
+getLocalStorage();
+const inputTask = document.querySelector('.input-task');
+const toDoList = document.querySelector('.to-do-list');
+
+document.querySelector('.add-task').addEventListener('click', () => {
+  if (inputTask.value !== '') {
+    addToDoTask(inputTask.value);
+    inputTask.value = '';
+  }
+});
```

Comment:
- [ ] Nice for trying to solve the problem that the previous reviewer asked but is it still it happened when I delete a task the data in the local storage doesn't get updated  I suggest when you call the delete function  to delete a task you should immediately call the a function that update the localstorage